core/data/src/main/java/kr/co/cotton/core/data/news/di/NewsModule.kt

```diff
@@ -0,0 +1,36 @@
+package kr.co.cotton.core.data.news.di
+
+import dagger.Binds
```

Comment:
🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between